sf-docker/special/sf-with-keycloak/keycloak-server/docker-compose.yml

```diff
@@ -1,17 +1,21 @@
 version: '3.7'
 services:
 
-  keycloak-server.10.11.81.42.nip.io:
-    image: quay.io/keycloak/keycloak:20.0.0
+  # Keycloak-server
+  #
+  # We use `keycloak-server.127.0.0.1.nip.io` as name of our container, because we need same
+  # hostname for this server from inside of docker-compose and also from the outside.
+  #
+  # Note: alternative wildcard DNS service, in case nip.io is down: http://xip.io
+  keycloak-server.127.0.0.1.nip.io:
```

Comment:
I think this hack will not work when accessing the server from other machine.

What service will access keycloak from within this docker network?

Reply:
Current setup is the only one that works for local and remote access

Reply:
i tested it localy and it seems we not need to have keycloak-server.127.0.0.1.nip.io (it is working with keycloak-server as name as well)